src/librbd/librbd.cc

```diff
@@ -6352,6 +6352,47 @@ extern "C" ssize_t rbd_aio_compare_and_write(rbd_image_t image, uint64_t off,
   return 0;
 }
 
+extern "C" ssize_t rbd_aio_comparev_and_writev(rbd_image_t image,
+                                               uint64_t off, size_t len,
+                                               const struct iovec *cmp_iov,
+                                               int cmp_iovcnt,
+                                               const struct iovec *iov,
+                                               int iovcnt,
+                                               rbd_completion_t c,
+                                               uint64_t *mismatch_off,
+                                               int op_flags)
+{
+  librbd::ImageCtx *ictx = (librbd::ImageCtx *)image;
+  librbd::RBD::AioCompletion *comp = (librbd::RBD::AioCompletion *)c;
+  tracepoint(librbd, compare_and_write_enter, ictx, ictx->name.c_str(),
```

Comment:
```suggestion
  tracepoint(librbd, aio_compare_and_write_enter, ictx, ictx->name.c_str(),
```

Reply:
`aio_compare_and_write_enter/exit` does not exist. `rbd_aio_compare_and_write` uses `compare_and_write_enter/exit`. Do you want me to add it or stick with what I had?

Reply:
I'm not following...  I see
```
TRACEPOINT_EVENT(librbd, aio_compare_and_write_enter,
...
)
TRACEPOINT_EVENT(librbd, aio_compare_and_write_exit,
...
)
```
in `src/tracing/librbd.tp` and
```
extern "C" ssize_t rbd_aio_compare_and_write(rbd_image_t image, uint64_t off,
                                             size_t len, const char *cmp_buf,
                                             const char *buf, rbd_completion_t c,
                                             uint64_t *mismatch_off,
                                             int op_flags)
...
  tracepoint(librbd, aio_compare_and_write_enter, ictx, ictx->name.c_str(), ictx->snap_name.c_str(),
              ictx->read_only, off, len, cmp_buf, buf, comp->pc, op_flags);
...
  tracepoint(librbd, aio_compare_and_write_exit, 0);
```
in `src/librbd/librbd.cc`?

Reply:
Sorry my bad. Not sure where I was looking. Will change it.